metpy/calc/indices.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2008-2017 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Contains calculation of various derived indicies."""
+import numpy as np
+
+from .thermo import mixing_ratio, saturation_vapor_pressure
+from .tools import get_layer
+from ..constants import g, rho_l
+from ..package_tools import Exporter
+from ..units import check_units, units
+
+exporter = Exporter(globals())
+
+
+@exporter.export
+@check_units('[temperature]', '[pressure]', '[pressure]')
+def precipitable_water(dewpt, p, top=400 * units('hPa')):
+    r"""Calculate precipitable water through the depth of a sounding.
+
+    Default layer depth is sfc-400 hPa. Formula used is:
+
```

Comment:
I think this needs a `..math::`, since the doc build currently has warnings from this PR.